frontend/src/api/Authorization.ts

```diff
@@ -0,0 +1,7 @@
+import { createRequest } from '@/api/ApiProvider';
+import { METHOD } from '@/helpers/constants';
+import { type ICredentials } from '@/utils/interfaces';
+
+export const loginAction = async (data: ICredentials) => {

```

Comment:
better to call the variable authentificateUser

Reply:
Done